File: z-clients/javascript/src/models/authTokenCreateNamespaceIn.ts
// this file is @generated

export interface AuthTokenCreateNamespaceIn {
    name: string;
}

export const AuthTokenCreateNamespaceInSerializer = {
    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _fromJsonObject(object: any): AuthTokenCreateNamespaceIn {
        return {
            name: object['name'],
        };
    },

    // biome-ignore lint/suspicious/noExplicitAny: intentional any
    _toJsonObject(self: AuthTokenCreateNamespaceIn): any {
        return {
            'name': self.name,
        };
    }
}

File: z-clients/rust/src/models/auth_token_create_namespace_in.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, Deserialize, Serialize)]
pub struct AuthTokenCreateNamespaceIn {
    pub name: String,
}

impl AuthTokenCreateNamespaceIn {
    pub fn new(name: String) -> Self {
        Self { name }
    }
}
